packages/shared/utils/error.ts

```diff
@@ -26,6 +27,11 @@ export const isCommandError = (error: unknown, code: number): boolean => {
       // TODO [BAC-2330] The Analysis endpoint returns an error string instead of an error object.
       return error === "There are too many points to complete this operation";
     }
+  } else if (code === ProtocolError.LinkerDoesNotSupportAction) {
+    if (typeof error === "string") {
+      console.error("Unexpected error type encountered (string):", error);
+    }
+    return error === "The linker version used to make this recording does not support this action";
```

Comment:
Does this happen for this error code too? I thought it was specific to the way the Analysis stuff bundled along errors.

Reply:
It was partly copy-pasta, but yeah, by the time we get to here the error really is a raw string with that exact text

Reply:
Really! Even for this error code? 😮

Reply:
I think this had the unintended side effect of causing the log point panel to show "0" instead of the too-many-points error.

Reply:
Added logic to account for this in 0c9ac73ca98a3b9cb008ad5bf15d83ef054e6195

Seems like we should fix it higher up though